src/acquisition/ght/ght_update.py

```diff
@@ -348,15 +355,24 @@ def main():
   # decide what to update
   if args.location.lower() == 'all':
     locations = LOCATIONS
+  elif args.location.lower() == 'none':
+    locations = [NO_LOCATION_STR]
   else:
     locations = args.location.upper().split(',')
   if args.term.lower() == 'all':
     terms = TERMS
   else:
     terms = [args.term]
 
+  # country argument
+  # Check that country follows ISO 1366 Alpha-2 code. 
+  # See https://www.iso.org/obp/ui/#search.
+  if len(args.country) != 2:
+    raise Exception('country name must be two letters (ISO 1366 Alpha-2)')
+  country = args.country.upper()
+
```

Comment:
I recommend also checking that `location` "none" isn't allowed to be used when `country` is "US". (Otherwise I think it'll end up with a query for "US-none"?)

Reply:
Updated to allow users to do both (`location`="none", `country`="US") and (`location`="US", `country`="US").
This only applies to the US country; the second case is enabled for backwards compatibility.